hilti/runtime/src/fiber.cc

```diff
@@ -18,114 +18,91 @@
 #endif
 
 using namespace hilti::rt;
-using namespace hilti::rt::detail;
 
 const void* _main_thread_bottom = nullptr;
 std::size_t _main_thread_size = 0;
 
-extern "C" {
+// A dummy function which will be put on the bottom of each fiber's call stack. This function should never execute.
+[[noreturn]] static void fiber_bottom(Fiber* fiber, void* args) { abort(); }
```

Comment:
Since functions executing on these fibers always have the same signature I think we do not need this to have C linkage (and it seems to pass CI), but other than that I am not certain.

Reply:
I'm not quite sure who'd be doing the actual call into this function, if that's coming from the `fiber` C code (if it ever were called), I think it would need the "C" linkage.

Reply:
I changed the linkage.